packages/tasit-sdk/src/TasitSdk.test.js

```diff
@@ -145,7 +170,81 @@ describe("Decentraland", () => {
       );
       await executeOrder.waitForNonceToUpdate();
 
-      await confirmBalances(estate, [buyer.address, seller.address], [1, 0]);
+      await confirmBalances(
+        estate,
+        [buyer.address, seller.address],
+        [1, estateIds.length - 1]
+      );
+    });
+
+    describe("Decentraland tasit app test cases", () => {
+      beforeEach(
+        "create sell orders and remove wallets from contracts",
+        async () => {
+          marketplace.setWallet(seller);
+          for (let assetId of estateIds) {
+            const priceInWei = ONE.toString();
+            const expireAt = Date.now() + duration.years(1);
+            const createOrder = marketplace.createOrder(
+              estate.getAddress(),
+              assetId,
+              priceInWei,
+              expireAt,
+              gasParams
+            );
+            await createOrder.waitForNonceToUpdate();
+          }
+
+          mana.removeWallet();
+          land.removeWallet();
+          estate.removeWallet();
+          marketplace.removeWallet();
+        }
+      );
+
+      it("should list marketplace estates sell orders (without wallet)", async () => {
+        const estates = [];
+        for (let id of estateIds) {
+          estates.push({
+            id,
+            name: await estate.getMetadata(id),
+          });
+        }
+
+        expect(estates[0].name).to.equal("cool estate 0x1");
+      });
+
+      it("should get an estate info (without wallet)", async () => {
+        const estateId = 5;
+        const name = await estate.getMetadata(estateId);
+        expect(name).to.equal("cool estate 0x5");
```

Comment:
Maybe get an image URI too in the test?

Reply:
Only EstateRegistry.sol contract has a _setTokenURI internal function but it isn't called anywhere.

Reply:
I've just made some improvements to test case, I'm not sure if it is enough

Reply:
Ah okay, maybe there's storing the mapping to image data off-chain then and not using the full set of optional ERC721 features.

Reply:
Okay, I'll take a look!

Reply:
I think maybe if we add a comment in the code saying something like:
```javascript
// Note: If Decentraland were leveraging ERC721Metadata token URI's more,
// we would have done expectations on them in this test
```

Then we can resolve this thread